variables.tf

```diff
@@ -179,13 +179,13 @@ variable "networking_mode" {
 }
 
 variable "logging_enable_components" {
-  type        = string
-  description = "(Optional) The GKE components exposing logs. Supported values include: 'SYSTEM_COMPONENTS' and 'WORKLOADS'."
+  type        = set(string)
```

Comment:
The schema says this is a list of max items 1. `main.tf` treats it as a list of max items one, but should it? It seems confusing to do it this way as opposed to making the variable a string. What do you think @soerenmartius

Reply:
Hey, `enable_components` in the `logging_config` block accepts a list, the `logging_config` block itself can only exist once

Reply:
Oh i see, you're right!